archetypes/Browse/PoolsTable/index.tsx

```diff
@@ -161,6 +161,7 @@ const TokenRow: React.FC<{
             </TableRowCell>
             <TableRowCell>
                 <Button
+                    id="browse-mint"
```

Comment:
am not 100% sure what this is for but I approved

Reply:
element tracking id